tests/store/tracking/test_file_store.py

```diff
@@ -162,6 +172,141 @@ def test_list_experiments_paginated(self):
         if len(exps3) < 500:
             self.assertIsNone(exps3.token)
 
+    def test_search_experiments_view_type(self):
+        self.initialize()
+        experiment_names = ["a", "b"]
+        experiment_ids = self.create_experiments(experiment_names)
+        self.store.delete_experiment(experiment_ids[1])
+
+        experiments = self.store.search_experiments(view_type=ViewType.ACTIVE_ONLY)
+        assert [e.name for e in experiments] == ["a", "Default"]
+        experiments = self.store.search_experiments(view_type=ViewType.DELETED_ONLY)
+        assert [e.name for e in experiments] == ["b"]
+        experiments = self.store.search_experiments(view_type=ViewType.ALL)
+        assert [e.name for e in experiments] == ["b", "a", "Default"]
```

Comment:
If we don't specify the ordering, is the returned experiment order undefined ? If so we should sort the result and then compare with expected result, to make the test deterministic.

Reply:
We always sort experiments by experiment_id to break ties so the result is deterministic.

Reply:
Suggestion:

Shall we  break ties  by sorting by "create_time desc" ?
So that when no order clause specified, it will return latest created experiments. This is more reasonable when there're many experiments.

Reply:
ok. can we use `experiment_id desc` as break ties sorting ? New experiment ids are incremental.